AdlsSdkSamples/Program.cs

```diff
@@ -221,48 +178,36 @@ private static void RunFileTransfer(AdlsClient client)
             }
             var localDestFile = localFileTransferPath + @"\testlocalDownloadFile.txt";
             Console.WriteLine("Download of the uploaded file:");
-            client.BulkDownload(destFile, localDestFile, 1); // Source and destination could also be directories
+            using (FileStream stream = File.OpenWrite(localDestFile))
+            {
+                fileContentDown.Value.Content.CopyTo(stream);
+            }
             using (var stream = new StreamReader(File.OpenRead(localDestFile)))
             {
                 string line;
                 while ((line = stream.ReadLine()) != null)
-                {
+                {  
                     Console.WriteLine(line);
                 }
             }
             Directory.Delete(localFileTransferPath, true);
-            client.DeleteRecursive(remoteFileTransferPath);
+            client.DeleteDirectory(remoteFileTransferPath);
```

Comment:
Do you need to pass `recursive: true`?

Reply:
@tg-msft Yes, because the folder is not empty.